src/modules/dashboard/components/ActionsPage/FinalizeMotionAndClaimWidget/FinalizeMotionAndClaimWidget.tsx

```diff
@@ -194,6 +194,7 @@ const FinalizeMotionAndClaimWidget = ({
     const domainBalance = bigNumberify(domainBalanceData?.domainBalance || '0');
 
     if (
+      actionType !== ColonyMotions.MintTokensMotion &&
```

Comment:
I don't really understand how this fix works. 

We're filtering out the Mint Motion as an action from the _Finalize_ button validation logic, correct ?

That means that we're not really fixing the underlying problem, but just brute forcing it for the Mint Motion, where this problem shows up

I feel like there is something else at play here that is preventing the Motion to be finalized if you're minting more tokens than your current balance.

Reply:
It's just not applicable to Min tokens motion. We can finalise motion even if the balance is zero. There is nothing to prevent us from it. In other situations we need the token balance be more than what we need to pay/transfer. Otherwise we'll have a contract error - we can transfer/pay more than we our balance. Again for mint tokens it's not important as we don't need any balance to "transfer from".
Let me know if it's still not clear